docs/UserGuide.md

```diff
@@ -12,9 +12,36 @@ Interface (CLI) while still having the benefits of a Graphic User Interface (GUI
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick Start [coming soon]
+## Quick start
+
+1. Ensure you have Java `11` installed in your Computer. 
+   You can check the version by opening a command window, and running the `java -version` command.
+
+2. Download the latest `[CS2103-T16-2][Insurance4Insurance].jar` from [here](https://github.com/AY2021S1-CS2103-T16-2/tp/releases).
+
+3. Copy the file to the **empty folder** you want to use as the _home folder_.
+
+4. Launch the app by using the `java -jar` command (do not use double-clicking). 
```

Comment:
Same thing. The double clicking thing. If AB3 told us to launch using -jar then yeah sure

Reply:
Same for here, I followed the PE instructions